docs/installation-and-operations/misc/time-entries-corrupted-by-10-4/README.md

```diff
@@ -0,0 +1,124 @@
+# Fixing time entries corrupted by upgrading to 10.4.0
+
+<div class="alert alert-info" role="alert">
+
+**Note**: This guide only concerns installations having upgraded exactly to the OpenProject version 10.4.0. Installations having upgraded to 10.4.1 directly are not affected.
+
+</div>
+
+The migration scripts that ran as part of the OpenProject 10.4.0 upgrade include an unfortunate bug that leads to some installations suffering data loss. 
+Installations, that had time entry activities enabled/disabled per project, will have all their time entries assigned to a single time entry activity.
+
+This guide describes how to fix the data once this has happened.
+
+## Preconditions
+
+* A backup file of a database state prior to 10.4.0.
+* Credentials with the permission to create a database in the database server the OpenProject installation is running against.
+* Console access to the OpenProject server.
+
+## 1. Create a second database from the backup
+
+Backup scripts are by default created via the [built in OpenProject command](../../operation/backing-up). 
+When not following the default, the database or the OpenProject server itself may have been backed up. 
+This guide only covers the proceedings for the the built in backup command. 
+But the reader might deduce the steps neccessary to restore accordingly for a custom backup from this guide.
+
+As a result of this step, a second database, not the database OpenProject is currently connecting to, will contain the data of the backup.
+
+First, connect to the OpenProject server and get the necessary details about your current database:
+
+```bash
+$ openproject config:get DATABASE_URL
+#=> e.g.: postgres://<dbusername>:<dbpassword>@<dbhost>:<dbport>/<dbname>
+```
+
+Example:
+
+```bash
+$ openproject config:get DATABASE_URL
+postgres://openproject:L0BuQvlagjmxdOl6785kqwsKnfCEx1dv@127.0.0.1:45432/openproject
+```
+
+Using this connection string, the following command will create the database the backup will be restored to (named `openproject_backup` in this example):
+
+```bash
+$ psql "postgres://<dbusername>:<dbpassword>@<dbhost>:<dbport>/<dbname>" -c 'CREATE DATABASE <new_dbname>'
```

Comment:
@oliverguenther I am not sure whether the user will have enough permissions to create the database. Do you know?

Reply:
It might probably not have enough permissions. An alternative would be `su postgres -c createdb -O <dbusernamer> openproject_backup`.

Reply:
How is the initial database created by the setup `openproject configure` script?

Reply:
I tested on my local installation and it should be possible to use `psql $(openproject config:get DATABASE_URL) -c "CREATE DATABASE openproject_backup"` for the default installations. Best we note both options (above command + postgres user)

Reply:
The packager postgres addon is using `su postgres -c` to create the database: https://github.com/pkgr/addon-postgres/blob/master/bin/preinstall#L189